all/pom.xml

```diff
@@ -32,7 +32,7 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <group>${package.group}</group>
+                    <group>${project.artifactId}</group>
```

Comment:
Shouldnt this be project.groupId?

Reply:
I just removed the custom property and changed it to use the artifactId which was the value of that property. But it's certainly possible to review this.

Reply:
if the group id should be used it will be picked up in a feature major release